.travis.yml

```diff
@@ -23,4 +23,5 @@ script:
 - xcodebuild test -project ./Shell\ Sort/Tests/Tests.xcodeproj -scheme Tests
 - xcodebuild test -project ./Stack/Tests/Tests.xcodeproj -scheme Tests
 - xcodebuild test -project ./Longest\ Common\ Subsequence/Tests/Tests.xcodeproj -scheme Tests
-- xcodebuild test -project ./Bucket\ Sort/Tests/Tests.xcodeproj -scheme Tests
\ No newline at end of file
+- xcodebuild test -project ./Bucket\ Sort/Tests/Tests.xcodeproj -scheme Tests
+- xcodebuild test -project ./All-Pairs\ Shortest\ Paths/APSP/APSP.xcodeproj -scheme APSP
```

Comment:
do you prescribe a particular project/target/etc structure for submissions? Related to my question in #110 about dependency managers...